src/Pie.js

```diff
@@ -69,18 +72,18 @@ export default class PieChart extends Component {
     let radius = Math.min(x, y)
 
     let chart = Pie({
-      center: this.props.center || (this.props.options && this.props.options.center) || [0,0] ,
-      r: this.props.r || (this.props.options && this.props.options.r) || radius /2,
-      R: this.props.R || (this.props.options && this.props.options.R) || radius,
+      center: this.props.options.center || [0,0],
+      r: this.props.options.r || radius /2,
+      R: this.props.options.R || radius,
       data: this.props.data,
       accessor: this.props.accessor || identity(this.props.accessorKey)
     })
 
     let textStyle = fontAdapt(options.label)
 
     let slices = chart.curves.map( (c, i) => {
-      let fill = (c.item.color && Colors.string(c.item.color)) || this.color(i)
-      let stroke = Colors.darkenColor(fill)
+      let fill = this.color(i)
```

Comment:
This has reverted a latest master change.  Color from the data item should still take precedence over any other color setting.